Jenkinsfile

```diff
@@ -70,103 +55,146 @@ pipeline {
         }
       }
     }
-    stage('Dockerize SoQL Server PG') {
+    stage('Docker Build') {
```

Comment:
I added top level stages for "Docker Build" and "Publish".  This allows us to share the `when` condition and run them in parallel.